### DISCUSSION TITLE
Revert/Hard Spoon Documentation

### DISCUSSION THREAD

**Discussion by charmitro** [paritytech/substrate#12298] [Ideas]
I believe there should be a good documentation from Parity regarding reverting, hard spooning etc. of a bricked chain.

**Comment by bkchr** [comment]
Should probably be some issue here: https://github.com/substrate-developer-hub/substrate-docs

**Comment by kianenigma** [comment]
@andresilva can you perhaps share any already written notes / post-mortems about this, if any?
